activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-rest/src/test/java/org/activiti/cloud/services/modeling/rest/controller/ProjectControllerIT.java

```diff
@@ -466,7 +466,8 @@ public void should_throwSemanticModelValidationException_when_validatingProjectW
                             ModelValidationError::getDescription,
                             ModelValidationError::getValidatorSetName)
                 .containsOnly(tuple("No assignee for user task",
-                                    "One of the attributes 'assignee','candidateUsers' or 'candidateGroups' are mandatory on user task",
+                                    "One of the attributes 'assignee','candidateUsers' or 'candidateGroups' are mandatory on user task "
+                                            + "with id: 'sid-2582C722-01AF-413B-B358-4043E6428B77'",
```

Comment:
In addition to the `id`, I'd add the task name as well. It will allow locating the task directly in the diagram. What do you think?

Reply:
I think it's a good idea to improve the readability of no assignee errors. However, the name could be empty in the task so I added a bit of logic to handle empty names. How looks to you?

Reply:
Thanks. It looks good to me.